python/ray/train/constants.py

```diff
@@ -16,21 +16,20 @@
 )
 
 # Autofilled session.report() metrics. Keys should be consistent with Tune.
-TIMESTAMP = "_timestamp"
-TIME_THIS_ITER_S = "_time_this_iter_s"
-TRAINING_ITERATION = "_training_iteration"
+# The train provided `TIME_THIS_ITER_S` and `TIMESTAMP` will triumph what's
+# auto-filled by Tune session.
+TIMESTAMP = "timestamp"
+TIME_THIS_ITER_S = "time_this_iter_s"
 
-BASIC_AUTOFILLED_KEYS = {TIMESTAMP, TIME_THIS_ITER_S, TRAINING_ITERATION}
-
-DATE = "_date"
+TIME_TOTAL_S = "_time_total_s"
+# Note the following 3 are semantically different than what's auto-filled by Tune.
 HOSTNAME = "_hostname"
 NODE_IP = "_node_ip"
 PID = "_pid"
```

Comment:
Can we rename these constants to `WORKER_HOSTNAME`, `WORKER_NODE_IP` etc do make this clear?